test/parallel/test-child-process-bad-stdio.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+// Flags: --expose_internals
+const common = require('../common');
+const assert = require('assert');
+const cp = require('child_process');
+
+if (process.argv[2] === 'child') {
+  setTimeout(() => {}, common.platformTimeout(1000));
```

Comment:
The test works reliably for me (and runs faster) with the `setTimeout()` replaced with a `setImmediate()`. Are we sure that the timer is needed here? If so, any chance we can bring it down to 100ms instead of 1000ms?